src/dbnode/storage/series/series.go

```diff
@@ -60,11 +61,11 @@ type dbSeries struct {
 	opts Options
 
 	// NB(r): One should audit all places that access the
-	// series ID before changing ownership semantics (e.g.
+	// series metadata before changing ownership semantics (e.g.
 	// pooling the ID rather than releasing it to the GC on
 	// calling series.Reset()).
 	id          ident.ID
-	tags        ident.Tags
+	metadata    doc.Document
```

Comment:
metadata (`doc.Document`) also has the series `ident.ID` as well doesn't it?

Reply:
Will add a comment, they share the same bytes but we need to return it as ident.ID in a bunch of places unfortunately.